src/main/resources/templates/courseFormView.twig

```diff
@@ -1,32 +1,30 @@
 {% if courseCreationError %}
+    <script type="text/javascript" src="/static/js/jquery-3.2.1.min.js"></script>
```

Comment:
Is this necessary?.... There has to be a better way to do this

Reply:
I believe @brandon1024 's PR addresses this. Basically we are trying to use some javascript before it gets brought into the file, so here I just put everything in the file. It's ugly, but it works

Reply:
I guess that's fine then.